fractal_server/app/models/task.py

```diff
@@ -12,13 +12,15 @@
 class Task(_TaskBase, table=True):  # type: ignore
     id: Optional[int] = Field(default=None, primary_key=True)
     project_id: Optional[int] = Field(foreign_key="project.id")
+    # FIXME: is default={} needed for default_args and meta?
     default_args: Dict[str, Any] = Field(sa_column=Column(JSON), default={})
```

Comment:
Questions:
* Should both `default_args` and `meta` be `Optional`?
* Is `default={}` needed?

Reply:
@jacopo-exact

Reply:
probably better to have them optional, allow none and check a posteriori